.github/workflows/release-build.yml

```diff
@@ -31,3 +31,34 @@ jobs:
 
       - name: Publish
         run: make VERSION=${GITHUB_REF#refs/tags/v} publish
+
+        # Get release information to determine id of the current release
+      - name: Get Release
+        id: get-release-info
+        uses: bruceadams/get-release@v1.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Upload deployment YAML files as ZIP to GitHub release
+      - name: Upload Deployment YAML (zip)
+        id: upload-deployment-yaml-zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: https://uploads.github.com/repos/tomkerkhove/keda-sandbox/releases/${{ steps.get-release-info.outputs.id }}/assets?name=keda-1.4.1.zip
+          asset_path: keda-1.4.1.zip
```

Comment:
Is it intentional that the version (1.4.1) is hardcoded in here?

Reply:
as well as the uploads ulr contains `tomkerhove` repo :)

Reply:
Ha, nope 🙈

Reply:
I'll fix this asap but don't have any more time today/this week.

Seems like getting the version out of the tag and setting it as a variable is not a trivial thing 🙄

Reply:
I think https://github.com/kedacore/keda/pull/832 should simplify this for you. It's based on this [approach](https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/m-p/32167/highlight/true#M1027). I tried to use it before, but couldn't get it to work, though later realized that I was missing the env section on the tasks that need it set.

Reply:
That did the trick, thanks @ahmelsayed!